Configuration/AlCa/python/autoCond.py

```diff
@@ -16,9 +16,9 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
     'run2_design'       :   '106X_mcRun2_design_v6',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '106X_mcRun2_asymptotic_v6',
+    'run2_mc'           :   '106X_mcRun2_asymptotic_Candidate_2019_11_29_09_29_56',
     # GlobalTag for MC production (cosmics) with starup-like alignment and calibrations for Run2, Strip tracker in peak mode
-    'run2_mc_cosmics'   :   '106X_mcRun2cosmics_startup_deco_v3',
+    'run2_mc_cosmics'   :   '106X_mcRun2cosmics_startup_deco_Candidate_2019_11_29_09_32_59',
```

Comment:
@christopheralanwest @tocheng @tlampen as for #28355 I can upgrade candidate GT once the full GTs are available.